server/src/main/java/org/apache/iotdb/db/tools/TsFileResourcePrinter.java

```diff
@@ -37,7 +37,7 @@
   @SuppressWarnings("squid:S106")
   public static void main(String[] args) throws IOException {
 
-    String folder = "test";
+    String folder = "E:\\codestore\\incubator-iotdb2\\cluster\\target\\cluster-0.11.0-SNAPSHOT3\\data\\data\\sequence\\root.group_1\\0";
```

Comment:
change this as `data/data/sequence/root.group_1/0`

Reply:
Changed.